runtime/src/main/java/org/corfudb/protocols/wireprotocol/ICorfuPayload.java

```diff
@@ -75,6 +80,19 @@
                         x.readerIndex(x.readerIndex() + bytes);
                         return b;
                     })
+                    .put(StreamAddressRange.class, x ->
```

Comment:
don't use single letter variable

Reply:
Here I'll keep it consistent with the style throughout this file, otherwise need to change it everywhere.